sources/packages/web/src/components/students/modals/ConfirmEditApplication.vue

```diff
@@ -1,12 +1,19 @@
 <template>
   <modal-dialog-base title="Edit application" :showDialog="showDialog">
-    <template #content>
-      Editing your application will result in a new assessment, which could
-      delay your application. <strong>Are you sure you want to proceed?</strong>
+    <template #content v-if="props.isBeforeApplicationEdit">
```

Comment:
Props doesn't need to be returned and they are directly accessible in the template. 

![image](https://github.com/user-attachments/assets/e1846b6d-b0b3-4742-9760-f4c33226edeb)